pom.xml

```diff
@@ -25,9 +25,13 @@
     <junit.version>5.13.4</junit.version>
     <io.grpc.version>1.58.0</io.grpc.version>
     <mockito.version>5.18.0</mockito.version>
+    <sonar.plugin.version>5.1.0.4751</sonar.plugin.version>
```

Comment:
Sonar recommends pinning to a version.  If you don't they throw a warning in the run.